File: app/src/main/java/com/venditti/placares/helper/Base64Custom.java
package com.venditti.placares.helper;

import android.util.Base64;

public class Base64Custom {

    public static String codificarBase64(String text){
        return Base64.encodeToString(text.getBytes(), Base64.NO_WRAP).replace("(\\n|\\r)", "");
    }

    public static String decodificarBase64(String text){
        return new String(Base64.decode(text, Base64.DEFAULT));
    }

}

File: app/src/main/java/com/venditti/placares/helper/Base64Custom.kt
package com.venditti.placares.helper

import android.util.Base64

object Base64Custom {
    fun codificarBase64(text: String): String {
        return Base64.encodeToString(text.toByteArray(), Base64.NO_WRAP).replace("(\\n|\\r)", "")
    }

    fun decodificarBase64(text: String?): String {
        return String(Base64.decode(text, Base64.DEFAULT))
    }
}